sdk/tables/azure-data-tables/samples/tables_getting_started.cpp

```diff
@@ -2,34 +2,30 @@
 // Licensed under the MIT License.
 
 #include <azure/data/tables.hpp>
+#include <azure/identity.hpp>
 
 #include <cstdio>
 #include <iostream>
 #include <stdexcept>
 #include <thread>
 
-std::string GetConnectionString()
-{
-  const static std::string ConnectionString = "";
-
-  if (!ConnectionString.empty())
-  {
-    return ConnectionString;
-  }
-  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
-  if (!envConnectionString.empty())
-  {
-    return envConnectionString;
-  }
-  throw std::runtime_error("Cannot find connection string.");
-}
-
 using namespace Azure::Data::Tables;
 const std::string TableName = "sample1";
 
 int main()
 {
-  auto tableServiceClient = TableServiceClient::CreateFromConnectionString(GetConnectionString());
+  const std::string accountName{std::getenv("ACCOUNT_NAME")};
+  auto credential = std::make_shared<Azure::Identity::ChainedTokenCredential>(
+      Azure::Identity::ChainedTokenCredential::Sources{
+          std ::make_shared<Azure::Identity::AzurePipelinesCredential>(
```

Comment:
We almost certainly *don't* want PipelineCredential to be in samples, since it only ever works in an ADO pipeline.

Reply:
I agree.
We can't expect customers to use AzurePipelinesCredential in the "getting started" scenario.

The goal of using DAC in samples is to get identity out of their way as a concept to learn, when they care about a particular service SDK usage.